admin-data-explorer-play-server/src/main/scala/net/wiringbits/webapp/utils/admin/config/TableSettings.scala

```diff
@@ -20,5 +23,6 @@ case class TableSettings(
     referenceField: Option[String] = None,
     hiddenColumns: List[String] = List.empty,
     nonEditableColumns: List[String] = List.empty,
-    canBeDeleted: Boolean = true
+    canBeDeleted: Boolean = true,
+    primaryKeyDataType: String = "UUID"
```

Comment:
It would be ideal to create an ADT/enum instead, this way, we can't propagate invalid values, like:

```scala
sealed trait PrimaryKeyDataType extends Product with Serializable
object PrimaryKeyDataType {
  final case object UUID extends PrimaryKeyDataType
  final case object Serial extends PrimaryKeyDataType
}
```